models/articles.js

```diff
@@ -0,0 +1,21 @@
+const db = require("../db/connection");
+
+exports.selectAllArticles = () => {
+  const sql = 
```

Comment:
This query is wrong based on what it is returning in your test. Please see the previous comment and amend 👍